.github/CODEOWNERS

```diff
@@ -73,5 +73,12 @@
 # fast-web-utilities
 /packages/fast-web-utilities/ @janechu @chrisdholt @nicholasrice
 
-# Other
-.github/ @awentzel @janechu @chrisdholt @nicholasrice
\ No newline at end of file
+# Github management
+.github/ @awentzel @janechu @chrisdholt @nicholasrice
+
+# Circle Ci
+.circleci/ @awentzel @janechu @nicholasrice
```

Comment:
@awentzel can we also add @chrisdholt to this one as he does some work on the build/release side